system/jlib/jptree.ipp

```diff
@@ -403,15 +370,160 @@ private:
     void replaceSelf(IPropertyTree *val);
 
 protected: // data
+    // The packing (#pragma pack above the base class definition) is overridden because very large numbers of these objects are created.
+
     IPropertyTree *parent; // ! currently only used if tree embedded into array, used to locate position.
-    HashKeyElement *name;
     ChildMap *children;
     IPTArrayValue *value;
-    //The packing (#pragma pack) is overridden because very large numbers of these objects are created, and the
-    //following two members currently cause 8 bytes to be wasted.  Refactoring the contents of AttrMap into this
-    //class would allow the fields to pack cleanly.
-    AttrMap attributes;     // this has 2 "extra" bytes - which could pack into the space following the count
-    byte flags;             // this could also pack into the space following the count.
+    byte flags;
+    HashKeyElement *name = nullptr;
```

Comment:
New jira: Should open a new jira to review removing the packing from this structure - reordering the entries should enable the size to be the same, but access to be much more efficient.